### DISCUSSION TITLE
svg preview and video thumbnails

### DISCUSSION THREAD

**Discussion by timsofteng** [sxyazi/yazi#1090] [1. Q&A]
Hello. I have a simple questions. 

How to preview .svg files?

Is it possible to add video thumbnail as a preview option?

**Comment by sxyazi** [comment]
The latest main branch now supports SVG previews, https://github.com/sxyazi/yazi/pull/1050. 

For video previews, if you have installed `ffmpegthumbnailer` as per the [installation guide](https://yazi-rs.github.io/docs/installation) it will work.

**Comment by timsofteng** [comment]
@sxyazi 
nice!
When do you plan to release it?

**Comment by kqvanity** [comment]
@sxyazi  I've updated to the latest version `Yazi 0.3.3` and certain svg files still shown in black (all except for the ones under `/usr/share/icons/Adwaita/scalable`
I've also updated my personal conf as per [this](https://github.com/sxyazi/yazi/pull/1050/files#diff-62c176c03cb77cb11e287cf9fe420fe603abd55296017d7323328e37eb056818) change
The tasks pane shows a long list of `Run preloder magick with 1 target(s)`
